apps/meteor/server/services/video-conference/service.ts

```diff
@@ -686,13 +718,15 @@ export class VideoConfService extends ServiceClassInternal implements IVideoConf
 			},
 			providerName,
 		});
+
+		await this.runNewVideoConferenceEvent(callId);
+
+		await this.maybeCreateDiscussion(callId, user);
```

Comment:
Would something be affected here if we turn these into a p.all?

Reply:
Yes, the event needs to run first so that apps may assign an existing ID to new conferences (there's a pexip setting where discussions will be reused); Rocket.Chat only creates a discussion if the app didn't assign one during that event.